types/connect-trim-body/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "dt-header": false
```

Comment:
Please remove the patch version in your header and re-enable this rule. Thanks!

Reply:
Follow-up: I saw your comment in the original post. Unfortunately, a new patch version of this package needs to be released every time a change is made, so you cannot specify the patch version in the header.

Reply:
When you say, "the patch version in the header", are you referring to this code? 

```
// Type definitions for connect-trim-body 0.0.3
```

The `0.0.3` part? That version is referring to the [version of the npm package](https://github.com/samora/connect-trim-body/blob/master/package.json#L3), not these typings? 

Are you saying that we cannot merge this PR because the npm package `connect-trim-body` decided to use only patch versions for releases instead of a major/minor version? The project has not been updated in years. I doubt that I could get the author to release a new version. 

What do you recommend we do here?

Reply:
I'd suggest just leaving it as `// Type definitions for connect-trim-body 0.0`.
This package will then be initially published as `@types/connect-trim-body@0.0.0` and will bump the patch version with each change.

Reply:
Got it. Done.